.github/workflows/single-runner.yml

```diff
@@ -8,31 +8,39 @@ on:
         required: true
         default: dcp_mappluto
       latest:
+        type: boolean
         description: "Tag this version as latest (optional)"
         required: false
         default: "true"
       version:
         description: "The version of the dataset (i.e. 22v2, 21C) if needed (optional)"
         required: false
 
-
-
 jobs:
-  dataloading:
+  setup:
+    uses: ./.github/workflows/setup.yml
+  archive:
     runs-on: ubuntu-20.04
-    env:
-      AWS_S3_ENDPOINT: ${{ secrets.DO_S3_ENDPOINT }}
-      AWS_ACCESS_KEY_ID: ${{ secrets.DO_ACCESS_KEY_ID }}
-      AWS_SECRET_ACCESS_KEY: ${{ secrets.DO_SECRET_ACCESS_KEY }}
-      AWS_S3_BUCKET: edm-recipes
+    needs: setup
     steps:
-      - uses: actions/checkout@v2.3.4
-      - uses: NYCPlanning/action-library-archive@v1.1
-        name: Archive ${{ github.event.inputs.dataset }}
-        with:
-          name: ${{ github.event.inputs.dataset }}
-          latest: ${{ github.event.inputs.latest }}
-          version: ${{ github.event.inputs.version }}
-          s3: true
-          compress: true
-          output_format: pgdump shapefile csv
```

Comment:
What type of file formats does data library dump to Digital Ocean with the latest changes of removing the specification of the `output_format`?